src/components/PeekTools.ts

```diff
@@ -78,3 +80,17 @@ export function getSortedLaws(game: Game, sortCol: LawSortCols, sortDir: number,
     .sort((a, b) => genCompare(a[sortCol], b[sortCol]) * sortDir)
     .map((law) => ({ ...law, priority: law.priority.toFixed(2) }))
 }
+
+export type EventCol = "id" | "probability"
+
+export type EventRow = {
+  id: string
+  probability: string
+}
+
+export function getSortedEvents(store: Store, sortCol: EventCol, sortDir: number, allEvents: Event[]): EventRow[] {
+  return allEvents
+    .map((event) => ({ id: event.id, probability: event.probability(store) }))
+    .sort((a, b) => genCompare(a[sortCol], b[sortCol]) * sortDir)
+    .map((row) => ({ ...row, probability: (row.probability * 100).toFixed(2) }))
+}
```

Comment:
Das ist doch ziemlich ähnlich wie in der `EventMachine.ts`... ich überlege, ob wir die Funktion nicht einfach in die `store/getters.ts` rein nehmen sollten. Auch das `toFixed(2)` ist da nicht schädlich.

Reply:
...sind halt alle Events. Auch die, die momentan probability 0 haben. Und die Sortierung ist schon ziemlich speziell für Peek und EventRow.